float64[3] x
float64 t
float64[6] setpoint
